ansible/roles/vm_set/tasks/control_mux_simulator.yml

```diff
@@ -0,0 +1,43 @@
+
+- name: Start mux simulator
+  block:
+
+  - name: Install flask
+    pip: name=flask version=1.1.2 state=forcereinstall executable={{ pip_executable }}
+    become: yes
+    environment: "{{ proxy_env | default({}) }}"
+
+  - name: Copy the mux simulator to test server
+    copy:
+      src: mux_simulator.py
+      dest: "{{ root_path }}"
+      mode: 0755
+
+  - name: Set default mux_simulator_port
```

Comment:
We should add some check to prevent deploying multiple mux simulator server on a vmhost_server.